__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -191,229 +190,216 @@ describe('TaskDetails Page', () => {
         const errorElement = await screen.findByText('Something went wrong!');
         expect(errorElement).toBeInTheDocument();
     });
+
     it('Renders Task Started-on Date', async () => {
-        const { getByText } = renderWithRouter(
+        const { queryAllByText } = renderWithRouter(
```

Comment:
try using like screen.queryAllByText

Reply:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
updated, please re-check once